radionets/dl_framework/data.py

```diff
@@ -264,6 +264,9 @@ def load_data(data_path, mode, fourier=False, source_list=False):
         dataset containing x and y images
     """
     bundle_paths = get_bundles(data_path)
-    data = [path for path in bundle_paths if re.findall("samp_" + mode, path.name)]
+    data = np.sort(
+        [path for path in bundle_paths if re.findall("samp_" + mode, path.name)]
+    )
+    data = sorted(data, key=lambda f: int(''.join(filter(str.isdigit, str(f)))))
```

Comment:
What is the advantage of this?

Reply:
when comparing with the wsclean results for the paper the sorting was necessary